include/pluginlib/pluginlib_exceptions.h

```diff
@@ -39,63 +40,69 @@ namespace pluginlib
  * @class PluginlibException
  * @brief A base class for all pluginlib exceptions that inherits from std::runtime_exception
  */
-class PluginlibException: public std::runtime_error
+class PluginlibException : public std::runtime_error
 {
-  public:
-    PluginlibException(const std::string & error_desc) : std::runtime_error(error_desc) {}
+public:
+public:
```

Comment:
copy-n-paste ?

Reply:
Yeah, likely a problem from when I was de-conflicting the cherry-picking.